packages/components/package.json

```diff
@@ -91,13 +102,17 @@
     "@typescript-eslint/parser": "^6.8.0",
     "@vitejs/plugin-react": "^4.1.0",
     "@vitest/coverage-c8": "0.33.0",
+    "browserslist": "^4.23.0",
     "eslint": "^8.53.0",
     "eslint-plugin-storybook": "^0.6.15",
     "globby": "^14.0.0",
     "history": "^5.3.0",
     "jest-axe": "^8.0.0",
     "jsdom": "^22.1.0",
-    "parcel": "^2.10.3",
+    "lightningcss": "^1.23.0",
+    "lightningcss-cli": "^1.23.0",
+    "parcel": "^2.11.0",
+    "parcel-optimizer-react-client": "workspace:^",
     "postcss-scss": "4.0.9",
```

Comment:
Shall we remove this?

Reply:
why? It's what bundles it all :)

Reply:
doh now I got it, sorry. Thanks @ichim-david